frontend_v2/src/styles/base.scss

```diff
@@ -1011,165 +1011,7 @@ ul {
 .container {
   position: relative;
   z-index: 1;
-  padding: 15px 40px;
-}
-
-.container-top {
```

Comment:
We do not need different containers for different devices since the padding-top has been fixed to 64px. A lot of code refactoring follows.